nova_vm/src/ecmascript/builtins/data_view/abstract_operations.rs

```diff
@@ -0,0 +1,145 @@
+use crate::ecmascript::{
+    builtins::array_buffer::{array_buffer_byte_length, is_fixed_length_array_buffer, Ordering},
+    execution::Agent,
+};
+
+use super::DataView;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+struct ByteLength(pub usize);
+
+impl ByteLength {
+    pub fn value(value: usize) -> Self {
```

Comment:
**nitpick:** These can all be `const`, which may prod the compiler to inline them quicker and easier.